plugins/buildingplan/buildingplan.cpp

```diff
@@ -15,6 +15,12 @@
 #include "df/organic_mat_category.h"
 #include "df/world.h"
 
+#if (defined(__GNUC__) && !defined(__clang__) && __GNUC__ >= 14)
```

Comment:
could also be `#if __has_cpp_attribute(no_dangling)`, I think

Reply:
as per https://gcc.gnu.org/git/?p=gcc.git;a=blob;f=gcc/testsuite/g%2B%2B.dg/ext/attr-no-dangling4.C;h=e910723d9852f92914bf84130a80106f56e50ee1;hb=c7607c4cf18986025430ca8626abfe56bfe87106